micro-rdk-ffi/micrordklib-idf-component/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 idf_component_register(INCLUDE_DIRS "${CMAKE_CURRENT_SOURCE_DIR}")
 idf_build_set_property(INCLUDE_DIRECTORIES "${CMAKE_CURRENT_BINARY_DIR}/assets" APPEND)
 
-set(LIBMICRORDK_VERSION v0.3.0)
+set(LIBMICRORDK_VERSION v0.4.0)
```

Comment:
This is https://viam.atlassian.net/browse/RSDK-9497, undoing the pin previously set in https://viam.atlassian.net/browse/RSDK-9493